doc/nrf/known_issues.rst

```diff
@@ -1355,7 +1356,7 @@ OCX-168: Issues with reestablishing streams
 nRF Machine Learning
 ====================
 
-.. rst-class:: v2-2-0
+.. rst-class:: v2-3-0 v2-2-0
```

Comment:
btw. Could we also slightly improve this known issue?

```
   The MCUboot bootloader cannot access external flash because the chosen ``nordic,pm-ext-flash`` is not defined in MCUboot child image's devicetree.
   As a result, MCUboot cannot swap images that are received during the OTA update.

**Workaround**: Manually cherry-pick and apply the commit with the fix from the ``main`` branch (commit hash: ``f54f6bbd423b12a595e76425e688f034926b8018``) to fix the issue for nrf5340dk_nrf5340_cpuapp. The same fix needs to be applied for the Thingy:53 board.
```